src/content/_custom_n4_jira_script.js

```diff
@@ -0,0 +1,89 @@
+"use strict";
+
+const possibleButtons = document.getElementsByClassName("toggl-button");
+
+const element = document.querySelector(
+  '[data-component-selector="jira.issue-view.issue-details.full-size-mode-column"]'
+);
+
+togglbutton.render(
+  // The main "issue link" at the top of the issue.
+  "#jira-issue-header:not(.toggl)",
+  { observe: true },
+  function (elem) {
+    const issueWrapper = elem.querySelector(
+      '[data-testid="issue.views.issue-base.foundation.breadcrumbs.breadcrumb-current-issue-container"]'
+    );
+    let issueNumberElement;
+    let container;
+
+    if (issueWrapper) {
+      issueNumberElement = issueWrapper;
+      container = issueWrapper.parentElement;
+    } else {
+      container = elem.querySelector("[class^=BreadcrumbsContainer]");
+      issueNumberElement = container.lastElementChild;
+    }
+
+    if (container.querySelector(".toggl-button")) {
+      // We're checking for existence of the button as re-rendering in Jira SPA is not reliable for our uses.
+      if (process.env.DEBUG) {
+        console.info('🚫 "Jira 2020-01 issue detail" quit rendering early');
+      }
+      return;
+    }
+    try {
+      const link = togglbutton.createTimerLink({
+        className: "jira2018",
+        projectName: (projects, tasks) => {
```

Comment:
please make this a seperate named function outside of render function and pass it here